lib_i2s/api/i2s.h

```diff
@@ -209,7 +210,7 @@ void i2s_master(client i2s_callback_if i2s_i,
                 clock bclk,
                 const clock mclk);
 
-#if defined(__XS2A__) || defined(__DOXYGEN__)
+#if defined(__XS2A__) || defined(__XS3A__) || defined(__DOXYGEN__)
 
 /** I2S frame-based master component **for xCORE200 only**
```

Comment:
Comment out of date

Reply:
Don't understand?

Reply:
**for xCORE200 only** should be xcore.ai and xcore200 I assume